pulumi/policies/grapl/PulumiPolicy.yaml

```diff
@@ -0,0 +1,6 @@
+runtime:
+  name: python
+  options:
+    virtualenv: venv
```

Comment:
this should be a pants project, and therefore will not use a pulumi-supplied virtualenv